docs/modules/data-structures/pages/map-config.adoc

```diff
@@ -1,10 +1,27 @@
 = Configuring Maps
-:description: Map configuration is part of the overall Hazelcast server configuration. When you create your map, it "inherits" the configuration that's set up on the server. It is important to plan your map configuration ahead of time, as you cannot change the configuration of a map once it is created.
+:description: Map configuration is part of the overall Hazelcast server configuration. When you create your map, it "inherits" the configuration that is set up on the server. It is important to plan your map configuration ahead of time, as you cannot change the configuration of a map once it is created without restarting your entire cluster.
 [[map-config]]
 
 {description}
 
-The example configuration below shows the default
+== Hazelcast Map Configuration Defaults
+
+The hazelcast.xml/hazelcast.yaml configuration included with your Hazelcast distribution includes the following default settings for maps. 
+
+* One synchronous backup
+* Binary in-memory format
+* All other features are disabled
+
+If these settings are sufficient for your environment, you don't need to do anything else. When you create a map, the Hazelcast cluster will apply these settings to your map. 
+
+For details on map backups, refer to the xref:backing-up-maps.adoc[Making My Data Safe] section.
```

Comment:
instead of 'refer to', we usually just put 'see'. It keep things consistent. Also no need to add 'section'.